### PR TITLE
debug netlify site and deploy env vars

### PR DESCRIPTION
**Pull request by schehata**

### CODE REVIEW

**Comment by schehata** [comment]
verbal frontend-log:


```json
{
  "fields": {
    "args": [
      "/api/hello"
    ]
  },
  "level": "info",
  "message": "Hello from SWR",
  "vercel": {
    "environment": "production",
    "projectId": "prj_NHRfKvjJ98HnVZ1JQHocgGhqTLOZ",
    "region": "",
    "route": "",
    "source": "frontend-log",
    "userName": "islam-axiomco"
  }
}
```

**Comment by schehata** [comment]
verbal lambda-log:

```json
{
  "fields": {
    "url": "/api/hello"
  },
  "level": "info",
  "message": "Hello from function",
  "request": {
    "host": "axiom-local-next-8avdvo8f0-islam-axiomco.vercel.app",
    "ip": "***",
    "method": "GET",
    "path": "/api/hello",
    "region": "iad1",
    "scheme": "https",
    "statusCode": 200,
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.0 Safari/605.1.15"
  },
  "vercel": {
    "environment": "production",
    "projectId": "prj_NHRfKvjJ98HnVZ1JQHocgGhqTLOZ",
    "region": "iad1",
    "route": "/api/hello",
    "source": "lambda-log",
    "userName": "islam-axiomco"
  }
}
```

**Comment by schehata** [comment]
verbal edge-log:

```json
{
  "level": "warn",
  "message": "this is a log from edge function",
  "request": {
    "host": "axiom-local-next-8avdvo8f0-islam-axiomco.vercel.app",
    "ip": "105.40.15.43",
    "method": "GET",
    "path": "/api/edge-func",
    "region": "",
    "scheme": "https",
    "statusCode": 200,
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.0 Safari/605.1.15"
  },
  "vercel": {
    "environment": "production",
    "projectId": "prj_NHRfKvjJ98HnVZ1JQHocgGhqTLOZ",
    "region": "",
    "route": "/api/edge-func",
    "source": "edge-log",
    "userName": "islam-axiomco"
  }
}
```

File: README.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ For more information check out the [official documentation](https://axiom.co/doc
 - If you are using Vercel, make sure you have the [Axiom Vercel integration](https://www.axiom.co/vercel) 
 installed. On other platforms you must create an API token and set those environment variables:
 
+:warning: next-axiom is still experimental for non-vercel platforms and is subject to change.
```
Comment (bahlo) [inline README.md:25]:
```suggestion
:warning: next-axiom is still experimental for non-Vercel platforms and is subject to change.
```

**Review by bahlo — APPROVED** [review]

**Pull request merged**